src/main/resources/plugin.yml

```diff
@@ -1,4 +1,4 @@
 name: SuperJukebox
-authors: [Utarwyn, TCOOfficiall]
-version: 1.0.2
+author: Utarwyn & TCOOfficiall
```

Comment:
Could you use my format for **authors** key? It will be better I think!

Reply:
Ow sorry

Reply:
Shoud be fuxed